presto-main/src/test/java/com/facebook/presto/execution/TestQueryManagerConfig.java

```diff
@@ -21,6 +21,9 @@
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 
+import static com.facebook.presto.execution.QueryManagerConfig.MaterializeExchangesStrategy.ALL;
```

Comment:
`ALL` and `NONE` don't fit for static import since the name is too general :)

Reply:
I think the code like

`.setMaterializeExchangesStrategy(NONE)` is pretty readable.

Something like .set**MaterializeExchangesStrategy**(**MaterializeExchangesStrategy**.NONE) sounds like a tautology